datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -74,6 +74,7 @@
 
 # Metric types for which it's only useful to submit once per set of tags
 ONE_PER_CONTEXT_METRIC_TYPES = [aggregator.GAUGE, aggregator.RATE, aggregator.MONOTONIC_COUNT]
+TYPO_SIMILARITY_THRESHOLD = 0.85
```

Comment:
I'm curious in how sensitive it is, like how many false positive does it imply. This value can still be updated in the future, but we should keep an eye on it in the future releases

Reply:
Yep, I'm going to look into the sensitively more before marking this as ready to review. Right now it can detect a few letters misplaced, which could be too much.

Reply:
I bumped up the threshold to eliminate some real-world false positives